tvaultuiv2/src/views/private/safe/CreateSafe/index.js

```diff
@@ -79,6 +113,11 @@ const useStyles = makeStyles(() => ({
     justifyContent: 'center',
     overflowY: 'auto',
     padding: '10rem 0',
+    '@media (max-width: 767.95px)': {
```

Comment:
```suggestion
    '@media (max-width: 767.95px)': {
```

use media-query breakpoints from pre-defined breakpoints

Reply:
fixed